popup.html

```diff
@@ -86,5 +89,4 @@ <h3>NhacCuaTui Scrobbler</h3>
     </div>
   </div>
   </body>
-</html>
```

Comment:
Looks like the `loginPrompt` element got lost in the commits, please add it back in?